src/pages/Cart/CartMain/CartMain.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import CartBox from './CartBox/CartBox';
+import './CartMain.scss';
+
+const CartMain = ({ cartBox }) => {
+  return (
+    <div className="cart-main-container">
+      <div className="cart-box-top">
+        <h1 className="cart-comment">장바구니</h1>
+        <span className="cart-etc">-</span>
+      </div>
+      {cartBox &&
+        cartBox.map(({ id, name, type, unit, price }) => {
```

Comment:
```suggestion
      {cartBox?.cartBox.map(({ id, name, type, unit, price }) => {
```
optional chaining 문법 사용하면 좀 더 간결하게 표현할 수 있습니다! 검색해서 어떻게 동작하는지 찾아보고 이해한 뒤에 적용해주세요!